core/src/jmh/java/io/grpc/internal/SerializingExecutorBenchmark.java

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
This should probably go in the benchmarks directory under

benchmarks/src/jmh/java/io/grpc/internal/....

to match the other existing micro benchmarks.